src/site/content/en/progressive-web-apps/pwa-checklist/index.md

```diff
@@ -0,0 +1,477 @@
+---
+layout: post
+title: What makes a good Progressive Web App?
+authors:
+  - samrichard
+  - petelepage
+date: 2020-01-06
+updated: 2020-01-20
+description: |
+  What makes a good, or great Progressive Web App?
+tags:
+  - post
+  - progressive-web-apps
+draft: true
+---
+
+<!-- Disable heading-increment because it mucks with the Details widget -->
+<!--lint disable heading-increment-->
```

Comment:
Incremental headings is an [a11y concern](https://web.dev/heading-levels/)

@petele can you give me context on why incremental headings are messing with your design here (you explaining seems faster than me checking out the branch and editing the headers to see for myself)

Reply:
I believe it's because we have a Details shortcode that generates headings, see `{% DetailsSummary 'h3' %}` below. However our linter doesn't know anything about this shortcode as it only knows to lint markdown. So the linter was complaining that the headings were not incremental and we needed to tell it to chill.

Reply:
Welp if Rob is fine with it then I'm fine with it